keps/0002-controller-manager.md

```diff
@@ -269,20 +267,54 @@ Hopefully the content previously contained in [umbrella issues][] will be tracke
 
 [umbrella issues]: https://github.com/kubernetes/kubernetes/issues/42752
 
-## Implementation History
+### Graduation to Beta
+
+As part of the graduation to `stable` or General Availability (GA), we have set
+both process and technical goals.
+
+#### Process Goals
+
+- 
+
+We propose the following repository structure for the cloud providers which
+currently live in `kubernetes/pkg/cloudprovider/providers/*`
+
+```
+git@github.com:kubernetes/wg-cloud-provider
```

Comment:
What's this cloud provider? A sample implementation done by the working group?

Reply:
That is the tracking repository for the working group, @wlan0 . I think I added some hypothetical use cases mostly around moving issues away from `kubernetes/kubernetes` to be more narrowly scoped to the working group.  I was also thinking that the repo could be used to store any conformance tests or documentation needed by all Cloud Providers

Reply:
Working groups shouldn't have repos. They are temporary structures that shouldn't own long term artifacts. This should instead be owned by some SIG. If we can't find an owning SIG that is something to talk about in either sig architecture or the steering committee.

Reply:
Maybe cloud-provider-sample, similar to sample-apiserver?

If there's no better place, I recommend SIG API Machinery to own the sample code and docs for now. It's part of the general control-plane infrastructure, and API Machinery already owns many of the system's extension mechanisms. If we more appropriate home is found, we can transfer ownership later.